rocks/masakari-consolidated/rockcraft.yaml

```diff
@@ -0,0 +1,47 @@
+name: masakari-consolidated
+summary: Openstack masakari-consolidated
+license: Apache-2.0
+description: |
+  Ubuntu distribution of OpenStack masakari-consolidated
+version: '2024.1'
+
+# renovate: base: ubuntu:22.04@sha256:1b8d8ff4777f36f19bfe73ee4df61e3a0b789caeff29caa019539ec7c9a57f95
+base: ubuntu@22.04
+platforms:
+  amd64:
+
+package-repositories:
+  - type: apt
+    cloud: caracal
+    priority: always
+
+services:
+  wsgi-masakari-consolidated:
+    override: replace
+    command: apachectl -D FOREGROUND
```

Comment:
Actually, can you remove this service definition in the consolidated rock? thank you

Reply:
For sure. Keep it in engine and API?

Reply:
Can you rename the `wsgi-masakari-engine` into `masakari-engine`

Reply:
Do we want to leave "wsgi-masakari-api" named as-is?

Reply:
Yeah, this one is ok